### ISSUE TITLE
Wrong format on srt

### ISSUE THREAD

**Issue by niklasnilsson87** [goatandsheep/node-srt#20]
if I send in 
```
1
00:00:13,960 --> 00:00:18,416
I cannot believe it.
```

It throws an error, Isn't the correct format on srt with a " , " at millisecond?

**Comment by gregmsanderson** [comment]
Yep, you are right. I've just found the same. A valid .srt file fails to parse because of that.

_Guessing_ it's a result of the regex checking for a WebVTT-style time (so not allowing a comma):

https://github.com/goatandsheep/node-srt/blob/master/lib/parser.js#L14

**Comment by kms0219kms** [comment]
@niklasnilsson87 @gregmsanderson I just replaced all `,`s to `.`s to work around this issue on my fork, [waktaplay/node-srt-ts](https://github.com/waktaplay/node-srt-ts).

Can you try in this way?
(https://github.com/waktaplay/node-srt-ts/blob/v3.0.4/src/lib/parser.ts#L132,
https://github.com/waktaplay/node-srt-ts/blob/v3.0.4/src/lib/parser.ts#L144,
https://github.com/waktaplay/node-srt-ts/blob/v3.0.4/src/lib/parser.ts#L149)
